File: 250721/4번의 크기 비교/4-time-comparison.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intA = sc.nextInt();
        sc.nextLine();
        int intB = sc.nextInt();
        int intC = sc.nextInt();
        int intD = sc.nextInt();
        int intE = sc.nextInt();

        // 변수 정의
        int intResult;

        // 조건문
        intResult = intA > intB? 1 : 0;
        System.out.println(intResult);

        intResult = intA > intC? 1 : 0;
        System.out.println(intResult);

        intResult = intA > intD? 1 : 0;
        System.out.println(intResult);

        intResult = intA > intE? 1 : 0;
        System.out.println(intResult);
    }
}

File: 250721/4번의 크기 비교/4-time-comparison.py
# 입력
intA = int(input())
intB, intC, intD, intE = map(int, input().split())

# 변수 정의
intResult = 0

# 조건문
intResult = 1 if intA > intB else 0
print(intResult)

intResult = 1 if intA > intC else 0
print(intResult)

intResult = 1 if intA > intD else 0
print(intResult)

intResult = 1 if intA > intE else 0
print(intResult)